docs/source/train.rst

```diff
@@ -86,10 +86,10 @@ The command line provides the following arguments:
     The margin value in the score function. It is used by *TransX* and *RotatE*.
 
   - ``-de`` or ``--double_ent``
-    Double entitiy dim for complex number It is used by *RotatE*.
+    Double entitiy dim for complex number or canonical Polyadic. It is used by *RotatE* and *SimplE*.
 
   - ``-dr`` or ``--double_rel``
-    Double relation dim for complex number.
+    Double relation dim for complex number or canonical Polyadic.
```

Comment:
same here.
BTW does simplE need both -de and -dr?

Reply:
Yes. SimplE need -dr to double relationship so that the score is computed forward and backward: 
![image](https://user-images.githubusercontent.com/26889237/96819167-946ed800-1455-11eb-8fbe-6954e7d7e170.png)
where for each relation r there is a corresponding inverse relation r'